src/Cassandra/Data/Linq/CqlQuerySingleElement.cs

```diff
@@ -77,7 +77,7 @@ public new TEntity EndExecute(IAsyncResult ar)
         /// </summary>
         public new TEntity Execute()
         {
-            var queryAbortTimeout = GetTable().GetSession().GetConfiguration()?.ClientOptions.QueryAbortTimeout ?? ClientOptions.DefaultQueryAbortTimeout;
+            var queryAbortTimeout = GetTable().GetSession().Cluster.Configuration.DefaultRequestOptions.QueryAbortTimeout;
```

Comment:
hmm that makes sense, I'll create this comment on #445 since all of the work related to LINQ and Mapper for execution profiles is over there, this was just a quick fix.